templates/add_book.html

```diff
@@ -68,58 +68,6 @@
                 </div>
 
 
-<!-- Dynamiczny formularz który nie działa -->
```

Comment:
This commit has pretty long message attached ;) I think we should avoid this and stick to shorter messages.
Just for sure - did you speak about this removal with the author of this template? I am not sure who was responsible for creating this js stuff.

Reply:
Paweł created this template, I spoke with him on monday about it, because only in this template 'profile button' does not worked and when we delete this commented script everything work. 
Btw. Paweł is not using this commented script ;)

Reply:
Ok, thank you for your answer!